GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -129,6 +129,10 @@ public sealed partial class FormCommit : GitModuleForm
         private readonly TranslationString _commitCommitterInfo = new TranslationString("Committer");
         private readonly TranslationString _commitCommitterToolTip = new TranslationString("Click to change committer information.");
 
+        private readonly TranslationString _modifyCommitMessageButtonToolTip
+            = new TranslationString("If you change the first line of the commit message, Git will treat this commit as an ordinary commit." + Environment.NewLine
+                                    + "I.e. it may no longer be a fixup or an autosquash commit.");
```

Comment:
```suggestion
                                    + "i.e. it may no longer be a fixup or an autosquash commit.");
```

Reply:
How about this

> If commit message following squash! or fixup! will not match any commits in an interactive rebase this commit will not be marked for squashing or fixup.

The grammar or exact working can be changed but the idea is to not "invoke magic" by mentioning "regular" commits as if this is anything but. Fact is it's just a regular commit i.e. there is nothing in its metadata to say otherwise and only if you will proceed to do an interactive rebase the message _might_ have some effect.